src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -4,6 +4,10 @@
 import com.google.common.collect.Iterators;
 import com.google.common.primitives.Ints;
 import htsjdk.samtools.SAMFileHeader;
+import java.io.Closeable;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.function.Consumer;
```

Comment:
Unused imports (actually, this file does not need any change anymore)

Reply:
removed